hr_holidays_compute_days/tests/test_holidays_compute_days.py

```diff
@@ -75,7 +75,7 @@ def test_number_days_excluding_full_day_employee(self):
             'date_to': '1946-12-27 11:00:00',  # Friday
             'holiday_status_id': self.holiday_type.id,
             'employee_id': self.employee.id,
-        })
+        }).with_context(test_full_days=True)
```

Comment:
Instead of here, add the context in the object `self.HrHolidays` for having it everywhere, and no need to put it in any other place.

Reply:
Done!